### ISSUE TITLE
goss --format junit errors

### ISSUE THREAD

**Issue by aelsabbahy** [aelsabbahy/goss#134]
See embrasure/alpine-hubot-slack#3

@mika Any harm in changing the following lines in [junit.go](https://github.com/aelsabbahy/goss/blob/master/outputs/junit.go#L17-L19)?

from:
```
location, _ := time.LoadLocation("Etc/UTC")
timestamp := time.Now().In(location).Format(time.RFC3339)
```
to:
```
timestamp := time.Now().Format(time.RFC3339)
```

**Comment by mika** [comment]
Hi, no objections at all, just verified + it works as intended for me, so LGTM! Thanks!

**Comment by aelsabbahy** [comment]
Thanks for the quick response. Released in `v0.2.3`
